File: src/main/java/com/aquasecurity/plugins/trivy/model/Nvd.java
package com.aquasecurity.plugins.trivy.model;

import com.fasterxml.jackson.annotation.JsonProperty;

public class Nvd {
    @JsonProperty("V2Vector")
    public String v2Vector;
    @JsonProperty("V3Vector")
    public String v3Vector;
    @JsonProperty("V2Score")
    public double v2Score;
    @JsonProperty("V3Score")
    public double v3Score;
}

File: src/main/kotlin/com/aquasecurity/plugins/trivy/model/Nvd.kt
package com.aquasecurity.plugins.trivy.model

import com.fasterxml.jackson.annotation.JsonProperty

class Nvd {
    @JsonProperty("V2Vector")
    var v2Vector: String? = null

    @JsonProperty("V3Vector")
    var v3Vector: String? = null

    @JsonProperty("V2Score")
    var v2Score: Double = 0.0

    @JsonProperty("V3Score")
    var v3Score: Double = 0.0
}